tests/cmd/stability/main.go

```diff
@@ -124,29 +127,40 @@ func run() {
 	oa.CleanOperatorOrDie(ocfg)
 	oa.DeployOperatorOrDie(ocfg)
 
+	crdUil := tests.NewCrdTestUtil(cli, kubeCli, asCli, false)
```

Comment:
s/crdUil/crdUilt`

I guess we should use `Util` here, which is short for `utility`